CMakeLists.txt

```diff
@@ -54,6 +54,7 @@ endif()
 
 # add directory with library
 add_subdirectory(lib)
+add_subdirectory(tools)
```

Comment:
unsure about this one if added as is. will check. I'm worried that everyone will need to have SFML in order to build __the whole__ project (meaning, solver as well). If so, let's put it under if-clause. I can do it

Reply:
Fixed this one